target/linux/ipq40xx/files/drivers/net/phy/ar40xx.c

```diff
@@ -2036,9 +1832,7 @@ static int ar40xx_probe(struct platform_device *pdev)
 	swdev->vlans = AR40XX_MAX_VLANS;
 	swdev->ports = AR40XX_NUM_PORTS;
 	swdev->ops = &ar40xx_sw_ops;
-	ret = register_switch(swdev, NULL);
-	if (ret)
-		goto err_unregister_phy;
+	register_switch(swdev, NULL);
```

Comment:
Is it intentional to not check the return value here?

Reply:
It's not, I forgot to add the check.
Its added now.